yaetos/kafka_utils.py

```diff
@@ -55,7 +55,7 @@ def validate(self, message):
         try:
             jsonschema.validate(instance=message, schema=self.__schema)
             return True
-        except jsonschema.exceptions.ValidationError, e:
+        except(jsonschema.exceptions.ValidationError, e):
```

Comment:
👍

Reply:
You can merge this one already @arthurprevot  I will be doing a new one with some improvements (at least from my point of view), thanks!

Reply:
ok, good, merging